### PR TITLE
[DEX-26] Modernize jsbin sample

### PR DESCRIPTION
**Pull request by artismarti**
## Description

This work changes the following:
- Replaces `var` with `const` in code sample that users see when they finish the registration process.
- Adds comments about API key and channel name.
- Replaces the old Ably logo.

### Jira ticket:
* [DEX-26](https://ably.atlassian.net/browse/DEX-26)

## Review
Existing Version:
image

Changed Version:
image

### CODE REVIEW

File: content/code/website-examples/home-page.code [hunk 1]
```diff
@@ -1,6 +1,13 @@
 [--- Javascript ---]
-var ably = new Ably.Realtime('{{API_KEY}}');
-var channel = ably.channels.get('{{RANDOM_CHANNEL_NAME}}');
+/**
+ * This is a DEMO API KEY. Your own API key is within your app in the dashboard: 'https://ably.com/accounts/' .
```
Comment (m-hulbert) [inline content/code/website-examples/home-page.code:3]:
```suggestion
 *  Note: this example uses a demo API key. Your own API keys are available in your account dashboard: 'https://ably.com/accounts/'.
```

File: content/code/website-examples/home-page.code [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+ */
+const ably = new Ably.Realtime('{{API_KEY}}');
+
+/**
+ * This is a random channel name, update it to your channel name if testing it on an Ably App.
```
Comment (m-hulbert) [inline content/code/website-examples/home-page.code:8]:
```suggestion
 * Note: this example uses a random channel name. You can use your own channel name when running this code sample against an Ably app.
```

File: data/jsbins.yaml [hunk 3]
```diff
@@ -53,6 +51,8 @@ jsbin_hash:
   EZuljFh/oQj0cOkrUZpU7XoBjQM=: alohep
   aCl/jytvZrNnguSEukgdTXNIja4=: epovij
   MpDgpT2wvuGs712waej3+FufE7A=: oxewul
+  EjhhGNY3f3dw55pyTen7hGEhUow=: ohidar
```
Comment (m-hulbert) [inline data/jsbins.yaml:54]: I'm not sure why this JSBin was updated but it looks identical to its previous version.

**Review by m-hulbert — CHANGES_REQUESTED** [review]
The example config file shouldn't be deleted as it's referenced in the README. 

I added a couple of suggested tweaks on the code comments, but feel free to ignore these if you disagree. There's also a JSBin that was updated to a new version, but it looks identical and only appears to be used in the formatting guide as an example.

File: data/jsbins.yaml (hunk 3 above)
Comment (artismarti) [reply data/jsbins.yaml:54]: I accidentally changed the wrong code sample file & reverted the changes. Could that be it?

Do I need to change this?

**Comment by artismarti** [comment]
> The example config file shouldn't be deleted as it's referenced in the README.
> 
> I added a couple of suggested tweaks on the code comments, but feel free to ignore these if you disagree. There's also a JSBin that was updated to a new version, but it looks identical and only appears to be used in the formatting guide as an example.

Thanks @m-hulbert 

I've made the requested changes! :)

File: data/jsbins.yaml (hunk 3 above)
Comment (m-hulbert) [reply data/jsbins.yaml:54]: Yes, that will what triggered it. It should be fine though - just wanted to check incase it was something you'd meant to change.

**Review by m-hulbert — APPROVED** [review]

**Pull request merged**